### ISSUE TITLE
pip install fails on colab python 3

### ISSUE THREAD

**Issue by jonfleming** [amaiya/ktrain#130]
When I run 
`!pip3 install -q ktrain`

in a new colab notebook I get:
```Building wheel for ktrain (setup.py) ... done
  Building wheel for keras-bert (setup.py) ... done
  Building wheel for langdetect (setup.py) ... done
  Building wheel for seqeval (setup.py) ... done
  Building wheel for syntok (setup.py) ... done
  Building wheel for gast (setup.py) ... done
  Building wheel for keras-transformer (setup.py) ... done
  Building wheel for sacremoses (setup.py) ... done
  Building wheel for keras-pos-embd (setup.py) ... done
  Building wheel for keras-multi-head (setup.py) ... done
  Building wheel for keras-layer-normalization (setup.py) ... done
  Building wheel for keras-position-wise-feed-forward (setup.py) ... done
  Building wheel for keras-embed-sim (setup.py) ... done
  Building wheel for keras-self-attention (setup.py) ... done
ERROR: tensorflow-probability 0.10.0rc0 has requirement gast>=0.3.2, but you'll have gast 0.2.2 which is incompatible.
```

I had no problem running this a few days ago.  Doing some googling I found something that works:

`import sys`
`!{sys.executable} -m pip install -q ktrain`

**Comment by amaiya** [comment]
Thanks for letting me know.  Note that, despite the error, `ktrain` should still works.  I tested it using a DistilBert model and everything trained as normal despite the ERROR.

The error may have something to do with the fact that Colab is now using TensorFlow 2.2.0-rc3 for some reason.  `ktrain` currently uses TensorFlow 2.1.0, which is the stable version on PyPi.

The error may go away once TF 2.2.0 is officially released and we upgrade ktrain to use 2.2.0.

Thanks again.

**Comment by amaiya** [comment]
*ktrain* now supports TensorFlow 2.2.0 which is the currently the default on Google Colab.
